### PR TITLE
84 donotwatch frontend

### LINKED ISSUE #84: Add Frontend for DoNoWatchOptions
The Backend is now able to filter certain events but there is no front end in the options page

### PR DESCRIPTION
**Pull request by thervh70**
Will close #84 

Front end for the DoNotWatchOptions.
Please confirm functionality and check code.

### CODE REVIEW

**Comment by mpsijm** [comment]
Please document this more clearly, because I don't get it so users probably won't get it either... when I _enable_ the checkbox, the watching of hover/comments/etc will be _disabled_? 
In either case, when I check or uncheck the "hover" checkbox, events 202 and 203 (mouse-enter/-leave) are still logged...

**Comment by thervh70** [comment]
This feature is not working indeed. However that is not part of my task. I will discuss with @Exclaminator on how to fix this. I will decide on how to present the options to the user (texts on the options page) as soon as the logic behind it is working.

**Comment by mpsijm** [comment]
Fair enough, since the front-end works, I approve the functionality of this PR. I would still want to have a proper documentation with the checkboxes though.

**Comment by mpsijm** [comment]
Fair enough, since the front-end works, I approve the functionality of this PR. Please add a TODO on the options page in that case, then this can be merged. :)

**Comment by MathiasMeuleman** [comment]
In my opinion that is the wrong way around. You should first decide how the user should be able to view and change the settings and after that you adapt the back-end to fit these elements. This means that the front-end should still be changed in this PR, preferably in such a way that either the negations completely disappear, or else in such a way that it is made very very clear it is a negation ('Feature Selection' does not indicate a negation)

File: resources/options.html [hunk 1]
```diff
@@ -100,6 +100,36 @@
         </div>
     </div>
 
+    <!--TODO: Make text understandable to the user (will be done when backend is properly working.-->
+    <div class="col s12 m6">
+        <div class="card" id="donotwatch">
```
Comment (mdingjan) [inline resources/options.html]: Naming: shouldn't this ID be 'doNotWatch'?
Same holds for the IDs below, with the exception of the abbreviations.

**Comment by mdingjan** [comment]
Side note: once this PR is merged, I will change the changes so that they are compatible with my refactorings. ;)

**Comment by thervh70** [comment]
Done! This can be merged now.

**Comment by mdingjan** [comment]
I approve the changes for my feedback.
When @MathiasMeuleman has approved the changes for his feedback as well, this PR can be merged.

File: resources/options.html [hunk 2]
```diff
@@ -100,30 +100,29 @@
...
-                        <input type="checkbox" id="dnwscreen">
-                        <label for="dnwscreen">Elements on the screen. </label>
+                        <input type="checkbox" id="dnwScreen" checked>
+                        <label for="dnwScreen">Elements on the screen. </label>
                     </li>
                     <li>
-                        <input type="checkbox" id="dnwhover">
-                        <label for="dnwhover">Elements I hover over with the cursor. </label>
+                        <input type="checkbox" id="dnwHover" checked>
+                        <label for="dnwHover">Elements I hover over with the cursor. </label>
```
Comment (mpsijm) [inline resources/options.html]: Silly thing... Shouldn't this be "mouse"? Cause the cursor is an indicator where you type XD

**Comment by mpsijm** [comment]
Can be merged now in my opinion as well, please check my last comment before merging :)

**Comment by MathiasMeuleman** [comment]
Thank you so very much.
Will merge :)

**Pull request merged**